content/sensu-go/5.19/installation/platforms.md

```diff
@@ -97,12 +97,12 @@ Docker images that contain the Sensu backend and Sensu agent are available for L
 
 [Binary-only distributions][4] that contain the Sensu backend, agent, and sensuctl tool are available in `.zip` and `.tar.gz` formats.
 
-| Platform & Version | `amd64` | `arm64` | `armv5` | `armv6` |`armv7` | `386` |
-|--------------------|---------|---------|---------|---------|--------|-------|
-| Linux              | {{< check >}}      | {{< check >}}     | {{< check >}}      | {{< check >}}      | {{< check >}}     | {{< check >}}    |
-| Windows            | {{< check >}}      |         |         |         |        | {{< check >}}    |
-| macOS              | {{< check >}}      |         |         |         |        |       |
-| Solaris            | {{< check >}}      |         |         |         |        |       |
+| Platform & Version | `386` |`amd64` | `arm64` | `armv5` | `armv6` |`armv7` | `MIPS` |
```

Comment:
MIPS is comprised of 4 architectures similar to arm. For consistency, instead of MIPS we should list:
* mips
* mipsle
* mips64
* mips64le

Reply:
Thank you, @amdprophet -- I was trying to sneak around adding 4 new columns to this table 😅 I will update it.